spaceship/lib/spaceship/connect_api/models/app.rb

```diff
@@ -144,6 +144,12 @@ def fetch_edit_app_info(client: nil, includes: Spaceship::ConnectAPI::AppInfo::E
         end.first
       end
 
+      def fetch_latest_app_info(client: nil, includes: Spaceship::ConnectAPI::AppInfo::ESSENTIAL_INCLUDES)
+        client ||= Spaceship::ConnectAPI
+        resp = client.get_app_infos(app_id: id, includes: includes)
+        return resp.to_models.first
+      end
```

Comment:
I’m curious if we need to do a sort here to first grab the `app_store_state` to grab the edit states first over the live states 🤔 Since I’m guessing we still want to give the edit states priority over the live ones (and we can’t fully trust the API). Thoughts?

Reply:
I initially thought this made sense 100%, but while implementing it, I thought maybe there are legit cases where the latest is not the edit version, but it's the one the user intends to fetch app info from 🤔 I'm not sure how the API sorts them  - it could be by most recently updated, most recently created, or by semantic versions, for instance. If it's by anything other than creation date (e.g. update date, or semantic version), our "assumption" that edit versions should take precedence over live ones isn't really true, I think.

Maybe we could leave it as is and trust the API - and in case it becomes a problem, we revisit? What do you think?
Although I feel your pain of working with undocumented that can't be trusted in the past, we're working with a documented and official set of APIs now, that _maybe_ we should be able trust that it have its inner logic updated silently? 😅  I tend to avoid putting extra business logic in the client/consumer app also. 

Anyways, if you still think we should add that logic here, I'm up for it. I propose this:

```ruby
def fetch_latest_app_info(client: nil, includes: Spaceship::ConnectAPI::AppInfo::ESSENTIAL_INCLUDES)
  client ||= Spaceship::ConnectAPI
  resp = client.get_app_infos(app_id: id, includes: includes)
  return resp.to_models.sort_by do |model|
    # Give preference to the edit app states over live ones
    Spaceship::ConnectAPI::AppInfo::AppStoreState::EDIT_APP_STATES.include?(model.app_store_state) ? 0 : 1
  end.first
end
```

Reply:
I think you are right! I think we can leave it like this for now 💪